### ISSUE TITLE
Copying weights 

### ISSUE THREAD

**Issue by rytp** [ioam/topographica#634]
HI,

I have 3 layers say A, B, C. B and C layers are of the same size. I am connecting layer A to layers B and C using the CFProjection. The weights between nodes in layer A and B are to be trained using the normalized hebbian mechanism. I now want the weights between nodes in layer A and C to be a replica of those between layers A and B. I have a different output function for layer C as compared to B.

Could someone suggest a way to copy these weights during every iteration of training.

**Comment by mjabri** [comment]
If input is same why not create a sheet A' and then connect A to A' using
CFProjection, and then A' to B and C with no weights, and just the output
functions of B and C as needed?

**Comment by rytp** [comment]
Thanks for the reply.

The issue with the suggestion is that the weights between A and A' will now be trained based on the values of the nodes in A and A', using the hebbian mechanism, and not the values of the nodes in A and B as required in the original question asked.

**Comment by mjabri** [comment]
I see, but then replicating the weights A -> B training using inputs to A and outputs of B will then not be applicable to A -> C if output functions of C are different than B.

**Comment by rytp** [comment]
Exactly. I do not want the weights between A and C to be trained. They should just be a copy of those between A and B.

**Comment by mjabri** [comment]
I think they are various ways yo can do this, and I think somethng (not exactly this) by creating a sheet (not projection sheet) that takes as port activities (before o/p functions) from other sheets. I passed the activities as through Dict in the ports.

**Comment by rytp** [comment]
Thanks. I will try that

**Comment by mjabri** [comment]
Good luck! Actually i found the example  ./examples/saccade_demo.ty helps understanding sending stuff to ports...
